plugins/org.jkiss.dbeaver.ext.altibase/OSGI-INF/l10n/bundle.properties

```diff
@@ -59,6 +59,12 @@ meta.org.jkiss.dbeaver.ext.altibase.model.AltibaseProcedure.schema.name
 meta.org.jkiss.dbeaver.ext.altibase.model.AltibaseProcedureParameter.length.name                          = Length
 meta.org.jkiss.dbeaver.ext.altibase.model.AltibaseProcedureParameter.parameterKind.name                   = Parameter Type
 meta.org.jkiss.dbeaver.ext.altibase.model.AltibaseProcedureParameter.scale.name                           = Scale
+meta.org.jkiss.dbeaver.ext.altibase.model.AltibaseProperty.attr.name									  = Attribute
```

Comment:
It looks like you have tab characters instead of spaces here. Not a big deal but ideally should be consistent with the rest of the file

Reply:
@ShadelessFox,

Thanks for your tip. I replaced all tab with spaces at the source code as you recommended :)